lib/internal/legacy/processbinding.js

```diff
@@ -33,4 +33,11 @@ module.exports = {
         ], key);
       })));
   },
+  natives() {
+    const { natives: result, configs } = internalBinding('builtins');
+    // Legacy feature: process.binding('natives').config contains stringified
+    // config.gypi
+    result.configs = configs;
+    return result;
```

Comment:
is the intention here to mutate the internal object?

if not:
```suggestion
    return { ...result, configs };
```

Reply:
Yes, the mutation is intentional. We do not use this internally.

Reply:
Can you add a comment about this?

Reply:
Done.